### PR TITLE
Updated text and button for the home component right side instruction…

### PR DESCRIPTION
**Pull request by bgsavage**
…s to support html.

image

### CODE REVIEW

File: webapp/src/main/webapp/src/assets/i18n/en.json [hunk 1]
```diff
@@ -36,8 +36,9 @@
       "title": "Access Assessment Results",
       "welcome": {
         "title": "Welcome!",
-        "sub-title": "Get the most from the Reporting System:",
-        "interpret-instructions": "Learn how to interpret the scores in this reporting tool:"
+        "user-guide-html": "<a href=\"{{userGuideUrl}}\" target=\"_blank\">The User Guide</a> describes features of the Smarter Balanced Reporting System and instructions for using each feature.",
+        "interpretive-guide-html": "<a target=\"_blank\" class=\"cursor-pointer\">The Interim Assessments Interpretive Guide</a> is designed to help educators, parents, and other stakeholders interpret interim assessments reports for the Smarter Balanced Reporting System.",
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: This one does not have a href yet as it should just show the "Coming soon" notification for now, right?  Though that's going away and might be a pita to implement a function call from the translate for something going away 9/15.

Comment (aaronsleeper) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: we have a link to a page that says "coming soon"?
are you saying it'll be launched on 9/15 and no longer coming soon?

Comment (esotrope) [inline webapp/src/main/webapp/src/assets/i18n/en.json]: You may want to just use ```href="#"```
Then you wont need the ```cursor-pointer``` css class

Comment (bgsavage) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: href="#" reloads the page, which is not what I want to happen.

Comment (bgsavage) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Yep, though I'm not sure what the plan is to redeploy on 9/15?  @malaffoon @jtreuting Will we have a hotfix on 9/15 to remove the notifications?

Comment (malaffoon) [reply webapp/src/main/webapp/src/assets/i18n/en.json]: Presumably yes.

**Pull request merged**